nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -387,6 +387,10 @@ in
                 Restart = "always";
                 Type = "simple";
               });
+
+              restartTriggers = [
```

Comment:
Are you sure this works? 
I believe `restartTriggers` should be an attribute of `service` (one level up), not of `serviceConfig`.

Reply:
You're right, it should probably be and attribute of `service` instead of `serviceConfig`.

How about I close this and someone merges #48658.